2996_reflection/reflection.md

```diff
@@ -4073,6 +4433,40 @@ consteval size_t bit_size_of(info entity);
 :::
 :::
 
+### [meta.reflection.result] Expression result reflection {-}
+
+::: std
+::: addu
+```cpp
+template <typename T>
+  consteval info reflect_value(T expr);
+```
+
+[#]{.pnum} *Mandates*: `T` is a structural type. `T` is not a reference type. Any subobject of the value computed by `expr` having reference or pointer type designates an entity that is a permitted result of a constant expression.
+
+[#]{.pnum} *Returns*: A reflection of the value computed by an lvalue-to-rvalue conversion applied to `expr`. The type of the reflected value shall be `T` if `T` is of class type, and otherwise the cv-unqualified version of `T`. The type of the reflected value shall not be an alias.
```

Comment:
As above. Also I don't think we need to say that `T` isn't an alias since it already cannot be. 
```suggestion
[#]{.pnum} *Returns*: A reflection of the value computed by an lvalue-to-rvalue conversion applied to `expr`. The type of the reflected value is `T` if `T` is of class type, and otherwise the cv-unqualified version of `T`.
```
Also, do we really need this to ever give a const value? Like is somebody actually going to write `reflect_value<T const>(something)`? Can we just make it always return cv-unqualified `T`?

Reply:
Ah, great catch! We talked through this a little while ago, and concluded that it will never be `T const`, even if you write that. Fixed, thanks.